src/olympia/pages/templates/pages/shield_study_9.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Get Synced</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_9/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Get Synced</h1>
+            <a href="https://addons.mozilla.org/firefox/downloads/latest/get-synced-v1-0/platform:3/addon-760109-latest.xpi?src=shield-study-9"><button class="button primary large">Get It Now</button></a>
```

Comment:
I don't think you can in pure HTML, you'd need to use our platform detection. But why does that extension have different files for each platform anyway? Does it interact with native specific stuff?